packages/components/modules/activity-log/ActivityLogComponent/index.tsx

```diff
@@ -0,0 +1,50 @@
+'use client'
+
+import { FC, useState } from 'react'
+
+import { Box } from '@mui/material'
+
+import { ActivityLogsFragment$key } from '../../../__generated__/ActivityLogsFragment.graphql'
+import ActivityUserSearchBar from '../ActivityUserSearchBar'
+import DateRangeChip from '../DateRangeChip'
+import EventFilterChip from '../EventFilterChip'
+import LogGroups from '../LogGroups'
+import { useActivityLogs } from '../graphql/queries/ActivityLogsFragment'
+import { ActivityLogComponentProps } from './types'
+
+const ActivityLogComponent: FC<ActivityLogComponentProps> = ({ queryRef }) => {
```

Comment:
atm we cant access any LogGroups props if we don't add it here, let's do it so we add an extra layer of customizability
```suggestion
const ActivityLogComponent: FC<ActivityLogComponentProps> = ({ queryRef, LogGroups = DefaultLogGroups, LogGroupsProps }) => {
```